src/rgw/rgw_op.cc

```diff
@@ -5863,6 +5863,12 @@ void RGWPutLC::execute(optional_yield y)
     ldpp_dout(this, 15) << "New LifecycleConfiguration:" << ss.str() << dendl;
   }
 
+  auto validate_only = s->info.env->get("HTTP_VALIDATE_ONLY");
```

Comment:
Is validate-only a standard header? Otherwise I think a prefix (I've used x-rgw- in StoreQuery) might be useful. I know there's some controversy about the 'X-' prefix, but that's to do with standardisation and we're not trying to do that.

Reply:
Sounds good will change it to that!

Reply:
Switched to `x-rgw-validate-only` see 173d310547ea18693f2aba1f21fc18acafa6a840

Reply:
Switched to `env->exists()`: 088ee99c90b726f77e149a6870cf32feb9c82b40